content/tutorials/spatial_wfs_services/index.Rmd

```diff
@@ -592,8 +535,101 @@ xy <- data.frame(id = c("loc1", "loc2"),
 
 xy %>%
   group_by(id) %>%
-  summarise(soil_map_data_extraction(x, y))
+  summarise(extract_soil_map_data(x, y)) %>%
+  knitr::kable()
+```
+
+
+## Example 5: pagination
+
+
+When a layer of a WFS service contains many thousands of features, oftentimes the WFS service will restrict the number of features that can be obtained with one request.
+This is done to optimize performance (responsiveness) of the WFS service: it is more efficient to transfer small packages of data and this is even more the case when multiple user requests are transmitted concurrently to the service.
+
+In this example we show how to deal with this situation.
+A technique called 'pagination' can be used to obtain all features one is interested by sending multiple requests to the server.
+The service is available from version '2.0.0' onwards.
+Servers running older versions may or may not have support for 'pagination'.
+
+We will use the 'Watervlakken' WFS service for this example.
+
+```{r}
+# this layer returns just 1000 features (for performance reasons)
+wfs <- "https://gisservices.inbo.be/arcgis/services/Watervlakken/MapServer/WFSServer?"
+url <- parse_url(wfs)
+url$query <- list(service = "wfs",
+                  version = "2.0.0", # facultative
+                  request = "GetFeature",
+                  typename = "Watervlakken:Watervlakken",
+                  srsName = "EPSG:31370",
+                  outputFormat = "GML32"
```

Comment:
Is the `outputFormat` argument actually needed in the examples in this tutorial? EDIT: except where you use `httr::write_disk()` (example 3). There's a default format per WFS which I think could work fine (I believe I tested it once, but not sure for all cases).

Reply:
Will do :+1: